trade-finance-manager-ui/templates/case/underwriting/lead-underwriter/assign-lead-underwriter.njk

```diff
@@ -3,6 +3,8 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle -%}Assign a lead underwriter{%- endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.